<?xml version="1.0"?>
<!--Projection PointCloud-->

<launch>
    <!--tf-->
    <!-- <node pkg="tf" type="static_transform_publisher" name="velodyne2realsense"  -->
    <!--       args="0.111393 -0.0103084 -0.0137764 0.0 0.0 0.0 velodyne camera_link 100" /> -->
    <!-- <node pkg="tf" type="static_transform_publisher" name="camera_link2camera_color_frame"  -->
    <!--       args="0.000 0.015 0.000 0.001 -0.001 -0.017 camera_link camera_color_frame 100"/> -->
    <!-- <node pkg="tf" type="static_transform_publisher" name="camera_color_frame2camera_color_optical_frame"  -->
    <!--       args="0.000 0.000 0.000 -1.571 0.000 -1.571 camera_color_frame camera_color_optical_frame 100"/> -->

    <!--velodyne-->
    <include file="$(find velodyne_pointcloud)/launch/32e_points.launch" />

    <!-- republish -->
    <!-- <node name="republish" type="republish" pkg="image_transport" output="screen" -->
    <!--       args="compressed in:=/occam/image0 raw out:=/occam/image0" /> -->

    <!--coloring pointcloud-->
    <node pkg="sensor_fusion" type="matching_points_image" name="matching_points_image">
        <remap from="/image" to="/occam/image0" />
        <remap from="/cinfo" to="/occam/camera_info0" />
        <remap from="/lidar" to="/velodyne_points" />
        <remap from="/projection" to="/occam_projection" />
    </node>

    <!--rviz-->
    <!-- <node pkg="rviz" type="rviz" name="rviz" args="-d $(find velodyne_camera_calibration)/config/projection_pointcloud.rviz"/> -->

    <!--rosbag play-->
    <!-- <node pkg="rosbag" type="play" name="play" args="$(find velodyne_camera_calibration)/bagfiles/data.bag" /> -->

</launch>
